tests/workflow/eden.eve-upgrade.tests.txt

```diff
@@ -0,0 +1,38 @@
+# Number of tests
+{{$tests := 18}}
```

Comment:
Please define 5 tests here. And check another workflows for the same issue.

Reply:
my bad, need to repush things